### PR TITLE
Msj 210 azure key vault

### PR DESCRIPTION
**Pull request by Ashishcloudmatos**

### CODE REVIEW

**Comment by dungdmatos** [comment]
Hey @AshishBotMantra could you remove the .ideal folder

**Review by dungdmatos — APPROVED** [review]

File: requirement.txt [hunk 1]
```diff
@@ -0,0 +1,14 @@
+azure-identity
```
Comment (rajesh-bagavathiyappan) [inline requirement.txt:1]: this file is not required

File: src/matos_azure_provider/provider.py [hunk 2]
```diff
@@ -17,7 +17,7 @@ class Provider(Connection):
         Connection (Class): base connection object
     """
 
-    def __init__(self, **kwargs) -> None:
+    def __init__(self, **kwargs):
```
Comment (rajesh-bagavathiyappan) [inline src/matos_azure_provider/provider.py:20]: retain the return type

`def __init__(self, **kwargs) -> None:`

File: src/matos_azure_provider/provider.py [hunk 3]
```diff
@@ -50,6 +50,7 @@ def fetch_discovery_details(rsc_type):
                     resources.append(result)
 
         service_map = self.service_factory.fetch_plugins()
+        print(service_map)
```
Comment (rajesh-bagavathiyappan) [inline src/matos_azure_provider/provider.py:53]: remove temporary code

**Review by rajesh-bagavathiyappan — CHANGES_REQUESTED** [review]

File: requirement.txt (hunk 1 above)
Comment (Ashishcloudmatos) [reply requirement.txt]: deleted plz review it now

**Review by rajesh-bagavathiyappan — APPROVED** [review]

**Pull request merged**